test/functional/p2p_feefilter.py

```diff
@@ -48,6 +37,12 @@ def on_inv(self, message):
             if (i.type == MSG_TX):
                 self.txinvs.append(hashToHex(i.hash))
 
+    def wait_for_invs(self, test_framework, invs_expected):
```

Comment:
perhaps call this `assert_invs_match` or `wait_for_invs_to_match`

Reply:
Done, I chose `wait_for_invs_to_match`.